### DISCUSSION TITLE
Can't upload a cart to the Tic-80 website

### DISCUSSION THREAD

**Discussion by mooncalf** [nesbox/TIC-80#2159] [Q&A]
I have all the metadata in, cover is snapped with F7 ... but the file is 208k and I saw someone else saying they couldn't upload anything bigger than 64k ... no response on that thread from a couple of years back.

Is there a limit to the cart size on the Tic-80 website?

Thanks.

Oh, here's the game on Itch if you want to check it out ... I never quite finished it, but it's playable. It's an homage to the Atari 2600 "Superman" of 1979.

[https://trepidgames.itch.io/s-man](https://trepidgames.itch.io/s-man)

**Comment by nesbox** [comment]
We have a similar issue https://github.com/nesbox/TIC-80/issues/2121 
Upload doesn't work for some pro carts, it needs investigation
